pcsx2/Config.h

```diff
@@ -549,7 +549,7 @@ struct Pcsx2Config
 		int UserHacks_TCOffsetX{0};
 		int UserHacks_TCOffsetY{0};
 		int UserHacks_CPUSpriteRenderBW{0};
-		TriFiltering UserHacks_TriFilter{TriFiltering::Automatic};
+		TriFiltering TriFilter{TriFiltering::Automatic};
```

Comment:
While I fundamentally agree that it's no longer a hack, changing the name here doesn't really gain much except breaking everyone's existing configuration.